### DISCUSSION TITLE
How to apply center crop in inference predict?

### DISCUSSION THREAD

**Discussion by papago2355** [open-edge-platform/anomalib#1222] [Q&A]
Hi. I used Padim and export to Openvino.
During inference predict, I found out center copped is not applied.
Is there a way to add center cropped in predict stage?
I can see center cropped is applied in metadata.json

**Comment by blaz-r** [comment]
This is part of pre-processing, so you'd need something like `openvino-inferencer` that also does that. You can see it [here](https://github.com/openvinotoolkit/anomalib/blob/main/tools/inference/openvino_inference.py)
